### ISSUE TITLE
[FEATURE]: Improve visibility of gaming data on interface

### ISSUE THREAD

**Issue by alvinlor2001** [COMP3122-2324-Project/Simpsyber#48]
In the game, I've noticed that developing different components has varying benefits, such as increasing the value of the product and generating more profits.
![2](https://github.com/COMP3122-2324-Project/Simpsyber/assets/30236827/5932bffd-99a2-4e6f-839e-8e00a66d319b)
However, it is a bit inconvenient to constantly click on the information button to check the actual benefit of each component. I would like to suggest a more user-friendly approach by making the changes visible directly on the screen. This could be achieved through visual indicators or icons that clearly show the impact of each component, similar to the image below.
![3](https://github.com/COMP3122-2324-Project/Simpsyber/assets/30236827/0e086662-443d-4afd-9aec-44a32a8f22b3)

I also recommend the same improvement to the staff ability interface. Currently, we have to click on "details" for each staff member to check their abilities, which can be tedious.
![4](https://github.com/COMP3122-2324-Project/Simpsyber/assets/30236827/c9db9cab-68a4-401b-aea4-2481445bbc57)
I recommend adding a small data block above each staff member that visually represents their unique abilities.
![5](https://github.com/COMP3122-2324-Project/Simpsyber/assets/30236827/32a94e17-ba1a-4729-96a2-31a034156fc1)
This would save time and make it easier for players to compare and strategize without having to click through each staff member individually.

**Comment by LeungYatOn** [comment]
Yea, when I was not very fimilar with the game, it is hard to memorialise the ability for the staffs. 

Add a star for recommendation maybe a good idea. 

![download](https://github.com/COMP3122-2324-Project/Simpsyber/assets/157469522/8d659501-ab02-4ca6-861a-aae8ef186144)

**Comment by CLee788** [comment]
I completely agree with your perspective on the use of indicators to enhance user-friendliness in the system. These indicators effectively communicate changes in the value of each function, eliminating the need for additional clicks. This solves the inconvenience of constantly clicking the information button to assess the actual benefits of each component. Such constant clicking can be time-consuming, disrupt the gameplay flow, and hinder effective comparison and strategizing. The plus signs you have displayed in the provided images are an excellent way to represent value changes.

Additionally, I would like to propose an improvement to the Staff Abilities interface. Similarly, to avoid the need for clicking on "details" for each staff member to check their abilities, I suggest adding a small data block in the select staff panel page. This data block would display the ability level of the staff in categories like "Design, Testing, Programming, and Infrastructure."

<img width="318" alt="image" src="https://github.com/COMP3122-2324-Project/Simpsyber/assets/157445841/b6b5c1ac-5cea-4d75-a1e1-dacdc967a4f6"> <img width="503" alt="image" src="https://github.com/COMP3122-2324-Project/Simpsyber/assets/157445841/a345e245-ab47-40cd-9b33-da6d839fd428">

Upon clicking the staff details page, the graph displaying staff abilities can be replaced with specific details for "Design, Testing, Programming, and Infrastructure." This way, users won't need to click on "details" for each individual staff member. Consequently, the "details" icon on the staff details page can be removed, as shown in the following images.

(Before)
<img width="468" alt="image" src="https://github.com/COMP3122-2324-Project/Simpsyber/assets/157445841/7ee9db75-6ea9-4966-8a7a-308ae94646b7">

(After)
<img width="500" alt="image" src="https://github.com/COMP3122-2324-Project/Simpsyber/assets/157445841/d34be3a8-170e-4e41-ad23-962df1066167">
